components/gnome-connection-manager/Makefile

```diff
@@ -0,0 +1,83 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+#
+# Copyright (c) 2015 Jim Klimov
+#
+
+include ../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		gnome-connection-manager
+COMPONENT_NAME_SHORT=	gcm
+COMPONENT_VERSION=	1.1.0
+COMPONENT_PROJECT_URL=	http://kuthulu.com/gcm
+COMPONENT_SRC=		$(COMPONENT_NAME_SHORT)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_NAME)_$(COMPONENT_VERSION)_all.tgz
+COMPONENT_ARCHIVE_HASH=	\
+    sha256:71a128676d16153c735499cec9f0a1af1e2dc1107ed5c57cea1994eadc120c4c
+COMPONENT_ARCHIVE_URL=	$(COMPONENT_PROJECT_URL)/$(COMPONENT_ARCHIVE)
+COMPONENT_BUGDB=	utility/$(COMPONENT_NAME)
+COMPONENT_SUMMARY=	Advanced SSH connections manager for X11
+COMPONENT_FMRI=		terminal/gnome-connection-manager
+COMPONENT_CLASSIFICATION=	Applications/System Utilities
+COMPONENT_LICENSE=	GPLv3
+COMPONENT_LICENSE_FILE=	$(COMPONENT_NAME).license
+
+include ../../make-rules/prep.mk
+include ../../make-rules/justmake.mk
```

Comment:
Why do you need it if you redefine $(BUILD_32) and $(INSTALL_32) targets? 
Perhaps, it's better to remove justmake.mk and simplify standard targets - e.g., remove PARFAIT rules, 
$(INSTALL_32) also seems overcomplicated. BTW, why use tar in one cases and $(CP) in other?
Just an idea: sometimes to avoid incorporating such rules in component Makefile you can deliver Makefile in files/Makefile and copy it to sources after extraction.